website/pages/docs/plugins/destinations/s3/overview.md

```diff
@@ -51,3 +51,11 @@ This is the (nested) spec used by the CSV destination Plugin.
 - `format` (string) (required)
 
   Format of the output file. `json` and `csv` are supported.
+
+- `include_headers` (boolean) (optional)
+
+  Whether the first line of the file should be the column headers. Default value is `false`.
```

Comment:
What's the reasoning behind defaulting to false?

Reply:
Backwards compatibility...

Reply:
Ah okay; if that's the main reason, I think we should consider doing a major bump instead. The data would be incredibly hard to use without headers 🤷

Reply:
I figured that by the time people realized they needed headers they would be comfortable enough to read the docs and make config changes to take advantage of this feature